package.json

```diff
@@ -94,7 +106,7 @@
     "agent"
   ],
   "license": "MIT",
-  "main": "dist/index.js",
+  "main": "dist/public/percy-agent.js",
```

Comment:
This should be `browser` and leave `main` in place for other SDKs like `@percy/puppeteer`

Like: https://github.com/percy/percy-puppeteer/blob/master/lib/index.ts#L2